client/extensions/woocommerce/woocommerce-services/views/label-settings/index.js

```diff
@@ -67,6 +67,7 @@ class AccountSettingsRootView extends Component {
 					setFormDataValue={ setValue }
 					selectedPaymentMethod={ ( formData || {} ).selected_payment_method_id }
 					paperSize={ ( formData || {} ).paper_size }
+					emailReceipts={ ( formData || {} ).email_receipts }
```

Comment:
Unfortunately this means that how to extract emailReceipts from formData is defined in two places (i.e. here and in the selector) (admittedly, not unlike the other fields) - maybe in a future PR/issue we could have this defined in a lib and then have one place to fix.